hub/src/server/utils.js

```diff
@@ -14,3 +17,46 @@ export function getDriverClass(driver: string) {
   }
 }
 
+
+export function readStream(stream: Readable, size: number): Promise<Buffer> {
```

Comment:
Is this method used anywhere besides the in-memory driver?  If not, then it should be moved to that driver's file.

Reply:
That seems to be the case. Will move it. 👍

Reply:
I believe this is going to get used by the revocation PR.